storage/client_event_test.go

```diff
@@ -32,7 +32,11 @@ import (
 )
 
 type storeEventReader struct {
-	perStoreFeeds map[proto.StoreID][]string
+	// If true, Update events will be recorded in full detail. If false, only a
+	// count of update events will be recorded.
+	recordUpdateDetail bool
+	perStoreFeeds      map[proto.StoreID][]string
+	perStoreUpdateC    map[proto.StoreID]int
```

Comment:
`C` suggests `chan` to me (as in `time.Ticker.C`); I'd call this `perStoreUpdateCount`